### ISSUE TITLE
9.0.1 build error on RHEL 6

### ISSUE THREAD

**Issue by devrimgunduz** [citusdata/citus#3153]
Hi,

I'm getting this while building Citus 9.0.1 on RHEL 6. Please note that 9.0.0 used to build fine:

```
gcc -Wall -Wmissing-prototypes -Wpointer-arith -Wdeclaration-after-statement -Wendif-labels -Wmissing-format-attribute -Wformat-security -fno-strict-aliasing -fwrapv -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64 -mtune=generic -fPIC -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64 -mtune=generic  -Wall -Wextra -Wno-unused-parameter -Wno-sign-compare -Wno-missing-field-initializers -Wno-clobbered -Wno-gnu-variable-sized-type-not-at-end -Wdeclaration-after-statement -Wendif-labels -Wmissing-format-attribute -Wmissing-declarations -Wmissing-prototypes -Wshadow -Werror=vla -I '/var/lib/pgsql/rpm11/BUILD/citus-9.0.1/src/include' -I'../../../src/include' -I. -I./ -I/usr/pgsql-11/include/server -I/usr/pgsql-11/include/internal -I/usr/include -D_GNU_SOURCE -I/usr/include/libxml2  -I/usr/include -I/usr/pgsql-11/include  -c -o progress/multi_progress.o progress/multi_progress.c
planner/shard_pruning.c:151: error: redefinition of typedef 'FunctionCallInfo'
/usr/pgsql-11/include/server/fmgr.h:38: note: previous declaration of 'FunctionCallInfo' was here
cc1: warning: unrecognized command line option "-Wno-gnu-variable-sized-type-not-at-end"
make[2]: *** [planner/shard_pruning.o] Error 1

```

Can you please take a look?

Thanks!

Regards, Devrim

**Comment by serprex** [comment]
This should be resolved once #3154 is released

**Comment by metdos** [comment]
> This is resolved once #3154 is released

Then, I suppose we can patch #3154 into the 9.0 release branch, @serprex, @JelteF?

**Comment by JelteF** [comment]
Sounds good

**Comment by hanefi** [comment]
I already created an issue for this build error: #3107

**Comment by marcocitus** [comment]
Fixed by #3154
